src/guide/components/slots.md

```diff
@@ -412,6 +412,36 @@ Passing props to a named slot:
 
 Note the `name` of a slot won't be included in the props because it is reserved - so the resulting `headerProps` would be `{ message: 'hello' }`.
 
+Also note that if you are mixing named scoped slots with the default scoped slot, you need to explicitly define the default template. Otherwise, it would hint that the data of the default scoped slot would be available in the other slots scopes.
+
+```vue-html
+<template>
+  <BaseLayout v-slot="{ message }">
+    <p>{{ message }}</p>
+    <template #footer>
+      <!-- message belongs to the default slot, and is not available here -->
+      <p>{{ message }}</p>
+    </template>
+  </BaseLayout>
```

Comment:
I think the two examples should use the same component name. As the preceding example also uses `MyComponent` I decided to go with that, but we could make them both `BaseLayout` if you prefer.

```suggestion
  <MyComponent v-slot="{ message }">
    <p>{{ message }}</p>
    <template #footer>
      <!-- message belongs to the default slot, and is not available here -->
      <p>{{ message }}</p>
    </template>
  </MyComponent>
```

Reply:
I'm sorry, it was clearly a failed copy paste from my local example, it's normal to have `MyComponent`